### PR TITLE
Android SDK 31

### PR DESCRIPTION
**Pull request by stackchain**
- Bump build to SDK 31 (Google play store is no longer accepting <31)

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -10,20 +10,32 @@
...
+def DEFAULT_NDK_API_LEVEL = 24
 
 def safeExtGet(prop, fallback) {
     rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
+apply plugin: "com.android.library"
```
Comment (stackchain) [inline android/build.gradle:25]:
```suggestion
```

File: react-native-haskell-shelley.podspec [hunk 2]
```diff
@@ -15,7 +15,7 @@ Pod::Spec.new do |s|
   # optional - use expanded license entry instead:
   # s.license    = { :type => "MIT", :file => "LICENSE" }
   s.authors      = { "emurgo" => "contact@emurgo.io" }
-  s.platforms    = { :ios => "9.0" }
+  s.platforms    = { :ios => "1111111111111111111111.0" }
```
Comment (stackchain) [inline react-native-haskell-shelley.podspec:18]:
```suggestion
  s.platforms    = { :ios => "11.0" }
```

**Comment by lisicky** [comment]
@stackchain Did you checked `example` directory ?

**Comment by stackchain** [comment]
chore #84

**Pull request closed without merging**